### ISSUE TITLE
What is the minimum amount of data need for training

### ISSUE THREAD

**Issue by slbinilkumar** [SeanNaren/deepspeech.pytorch#131]
I had tested with librispeech dev-clean and test-clean the result i am getting is like this.

Epoch: [41][27/27]      Time 0.486 (0.433)      Data 0.004 (0.016)      Loss 83.6796 (108.5965)
Training Summary Epoch: [41]    Average Loss 74.357
Validation Summary Epoch: [41]  Average WER 101.061     Average CER 55.925
Learning rate annealed to: 0.000006
Epoch: [42][1/27]       Time 0.617 (0.433)      Data 0.306 (0.017)      Loss 59.9648 (108.5526)
Epoch: [42][2/27]       Time 0.341 (0.433)      Data 0.004 (0.017)      Loss 45.3936 (108.4957)
Epoch: [42][3/27]       Time 0.349 (0.433)      Data 0.003 (0.017)      Loss 54.1301 (108.4467)
Epoch: [42][4/27]       Time 0.381 (0.433)      Data 0.003 (0.017)      Loss 60.3761 (108.4034)
Epoch: [42][5/27]       Time 0.360 (0.433)      Data 0.004 (0.017)      Loss 58.9576 (108.3590)
Epoch: [42][6/27]       Time 0.367 (0.433)      Data 0.017 (0.017)      Loss 59.8468 (108.3154)
Epoch: [42][7/27]       Time 0.402 (0.433)      Data 0.004 (0.017)      Loss 58.5789 (108.2707)
Epoch: [42][8/27]       Time 0.429 (0.433)      Data 0.014 (0.017)      Loss 69.7798 (108.2362)
Epoch: [42][9/27]       Time 0.378 (0.433)      Data 0.002 (0.017)      Loss 65.2217 (108.1977)
Epoch: [42][10/27]      Time 0.389 (0.433)      Data 0.003 (0.017)      Loss 72.8644 (108.1660)
Epoch: [42][11/27]      Time 0.402 (0.433)      Data 0.009 (0.017)      Loss 72.2627 (108.1339)
Epoch: [42][12/27]      Time 0.409 (0.433)      Data 0.005 (0.017)      Loss 65.3945 (108.0957)
Epoch: [42][13/27]      Time 0.451 (0.433)      Data 0.005 (0.017)      Loss 72.4847 (108.0639)
Epoch: [42][14/27]      Time 0.404 (0.433)      Data 0.004 (0.017)      Loss 77.0079 (108.0362)
Epoch: [42][15/27]      Time 0.429 (0.433)      Data 0.005 (0.017)      Loss 71.8329 (108.0040)
Epoch: [42][16/27]      Time 0.451 (0.433)      Data 0.008 (0.017)      Loss 79.4257 (107.9785)
Epoch: [42][17/27]      Time 0.439 (0.433)      Data 0.005 (0.017)      Loss 78.5896 (107.9524)
Epoch: [42][18/27]      Time 0.439 (0.433)      Data 0.006 (0.017)      Loss 84.4417 (107.9315)
Epoch: [42][19/27]      Time 0.465 (0.433)      Data 0.005 (0.017)      Loss 94.2493 (107.9193)
Epoch: [42][20/27]      Time 0.474 (0.433)      Data 0.007 (0.017)      Loss 83.3208 (107.8975)
Epoch: [42][21/27]      Time 0.468 (0.433)      Data 0.007 (0.017)      Loss 87.6460 (107.8795)
Epoch: [42][22/27]      Time 0.456 (0.433)      Data 0.006 (0.017)      Loss 91.2569 (107.8648)
Epoch: [42][23/27]      Time 0.457 (0.433)      Data 0.005 (0.016)      Loss 79.1524 (107.8394)
Epoch: [42][24/27]      Time 0.481 (0.433)      Data 0.007 (0.016)      Loss 91.6909 (107.8251)
Epoch: [42][25/27]      Time 0.485 (0.433)      Data 0.004 (0.016)      Loss 87.8362 (107.8075)
Epoch: [42][26/27]      Time 0.474 (0.433)      Data 0.004 (0.016)      Loss 85.2618 (107.7876)

**Comment by ryanleary** [comment]
You only trained on the development set?

**Comment by ryanleary** [comment]
I would also refer you to Table 10 of the DS2 paper (available here: https://arxiv.org/pdf/1512.02595.pdf). 

Their full training is on approximately 12000 hours of English. 120 hours works, but is pretty terrible. Realistically, you'll want at least 1000 hours for a reasonable system.

**Comment by slbinilkumar** [comment]
Thank you...
